src/test/java/rife/engine/TestEngine.java

```diff
@@ -44,6 +44,25 @@ public void setup() {
         }
     }
 
+    @Test
+    void testTomcatPlain()
+            throws Exception {
```

Comment:
`throws` statements are not indented in the style I use

Reply:
Please add your IDEA codestyle to git.

Reply:
I think I figured it out, I was not use per-project styles. It should now be updated.

Reply:
Fixed.